### PR TITLE
The most important change in mankind's history

### PR DESCRIPTION
**Pull request by DerZade**
This may sound dumb, but is from upmost importance. We are presented with a unique opportunity, which is naming a project [`meh`](https://www.urbandictionary.com/define.php?term=Meh) (aka the most awesome word in the whole universe) of course.

But there is still one problem to solve, we need a word starting with the letter H, which fits. Here are some examples we came up with:
- **M**ap **E**xport **H**elper
- **M**ap Export **H**andler
- **M**ap Export **H**uuuiiiii

PS: Of course merging this PR has to go alongside with renaming the repository to the name it's destined to be.

### CODE REVIEW

**Comment by nomisum** [comment]
map export häh?

File: src/main.cpp [hunk 1]
```diff
@@ -510,10 +510,10 @@ game_value exportMapCommand(game_state& gs, SQFPar rightArg) {
 }
 
 void intercept::pre_start() {
-    static auto grad_me_export_map =
+    static auto grad_meh_export_map =
         client::host::register_sqf_command("gradMeExportMap", "Exports the given map", exportMapCommand, game_data_type::BOOL, game_data_type::STRING);
```
Comment (TheWillard) [inline src/main.cpp]: gradMe**h**ExportMap?

**Comment by Synch2k** [comment]
Maybe something a little more extravagant? Map Export _Hero_ 🌈

**Comment by Fusselwurm** [comment]
> Maybe something a little more extravagant? Map Export _Hero_ 🌈

👍 perfect combination of highest aspiration and bitter disillusionment. meh hero!

File: src/main.cpp (hunk 1 above)
Comment (DerZade) [reply src/main.cpp]: Far better than `grad_map_export_export_map` 😅

Comment (DerZade) [reply src/main.cpp]: Fixed in https://github.com/gruppe-adler/grad_me/pull/1/commits/55cd8022cd3019e5acdb7594bb9b096573bfd8d7

**Pull request merged**